src/event.h

```diff
@@ -920,6 +925,24 @@ class event
                    > ()( calendar::turn, std::forward<Args>( args )... );
         }
 
+        template<event_type Type>
```

Comment:
This probably needs a comment explaining that it's essentially the same as `make` but with a runtime-defined number of arguments, and with all of the arguments given as strings which will be converted to the respective types.

Reply:
I'm not great with comments. If you have specific wording in mind, I can replace my blurb.